EventListener/VoteBlamerListener.php

```diff
@@ -26,9 +25,14 @@
 class VoteBlamerListener implements EventSubscriberInterface
 {
     /**
-     * @var SecurityContextInterface
+     * @var object
      */
-    protected $securityContext;
+    protected $authorizationChecker;
+
+    /**
+     * @var object
+     */
+    protected $tokenStorage;
```

Comment:
please make the new properties private (renaming the property is a BC break anyway for people extending it, so we can make things fully and switch the visibility)